usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -1049,6 +1096,8 @@ def initial_run(self, next_last_load):
                     FROM raw.transaction_normalized AS tn INNER JOIN raw.detached_award_procurement AS dap ON (
                         tn.transaction_unique_id = ucase(dap.detached_award_proc_unique)
                     )
+                    -- Want to exclude orphaned transactions, as they will not be copied into the int schema.
+                    WHERE tn.id NOT IN (SELECT transaction_id FROM temp.orphaned_transaction_info WHERE is_fpds)
```

Comment:
Should the `WHERE is_fpds` and `WHERE NOT is_fpds` be comparing to a boolean? Does this work or evaluate correctly if not? Just not sure if it comes up with falsey/truthy logic so long as there is any boolean value and it's not null.

Reply:
This column is declared as `NOT NULL`, so no need to worry about NULLs.